src/model/Gateway/Free.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Crm\PaymentsModule\Gateways;
+
+use Crm\ApplicationModule\Config\ApplicationConfig;
+use Crm\PaymentsModule\Repository\PaymentMetaRepository;
+use Nette\Application\LinkGenerator;
+use Nette\Http\Response;
+use Nette\Localization\ITranslator;
+use Omnipay\Omnipay;
+use Omnipay\PayPal\ExpressGateway;
+use Crm\PaymentsModule\Gateways\GatewayAbstract;
+
+class Free extends GatewayAbstract
```

Comment:
There are things, that are imported/initialized and never used. Please clear:

- All the unused imports
- All the things initialized in constructors and never used. If I look correctly, the `$paymentMetaRepository` shouldn't be here and you don't need the whole constructor to be overriden.